packages/core/lib/use-puck.ts

```diff
@@ -71,6 +73,12 @@ export const generateUsePuck = (store: PickedStore): UsePuckStore => {
     getItemBySelector: (selector) => getItem(selector, store.state),
     getItemById: (id) => store.state.indexes.nodes[id].data,
     getSelectorForId: (id) => getSelectorForId(store.state, id),
+    getParentById: (id) => {
+      const node = store.state.indexes.nodes[id];
+      const parentId = node.parentId;
+      if (parentId === null) return;
+      return store.state.indexes.nodes[parentId].data;
```

Comment:
```suggestion
      const node = store.state.indexes.nodes[id];
      
      const parentId = node.parentId;
      if (parentId === null) return;
      
      const parentNode = store.state.indexes.nodes[parentId];
      if (!parentNode) return;
      
      return parentNode.data;
```

Could you also move this function into a module under `/lib`, similar to how we’re handling `getSelectorForId`?